tests/test_parquet_export.py

```diff
@@ -36,7 +38,6 @@ async def test_parquet_conversion(
         records=generate_records(file_path, inspection, columns),
         columns=columns,
         output_name=None,

```

Comment:
Should help tests pass
```suggestion
        output_filename=None,
```